docs/Makefile

```diff
@@ -69,7 +67,7 @@ epub3: setup
 
 .PHONY: multiversion
 multiversion: setup
-	$(POETRY) run sphinx-multiversion $(SOURCEDIR) $(BUILDDIR)/dirhtml
+	$(POETRY) run sphinx-multiversion source $(BUILDDIR)/dirhtml
```

Comment:
Here's this "source" again. What is it, and why SOURCEDIR changed to "source" in this line, but a few lines below it's not changed?